lib/tasks/redis.rake

```diff
@@ -0,0 +1,22 @@
+require "redis"
+
+namespace :redis do
+  desc "Make redis a slave of production redis"
+  task :make_slave => :environment do
+    redis = Redis.new(url: ENV.fetch('REDIS_URL'), :driver => :hiredis)
```

Comment:
Hm, we may need to install the `hiredis` gem in order to use the driver, but last time I checked `hiredis` isn't supported on Heroku.

There's a buildpack to add it but that might be overkill:
https://elements.heroku.com/buildpacks/keiko713/heroku-buildpack-hiredis

I'd suggest holding off and seeing what kind of performance impact there is; if it's bad then we can consider adding it in.

Reply:
My bad, looks like things have changed and `hiredis` works on Heroku now!